jvm/core/src/main/kotlin/com/intuit/playerui/core/plugins/JSScriptPluginWrapper.kt

```diff
@@ -18,15 +18,10 @@ public abstract class JSScriptPluginWrapper(public val name: String, protected v
 
     final override lateinit var instance: Node protected set
 
-    protected val debugScript: String
-        get() = loadDebugScript() ?: script
-
-    private fun loadDebugScript(): String? = JSScriptPluginWrapper::class.java.classLoader.getResource(sourcePath?.substringBeforeLast(".") + ".dev." + sourcePath?.substringAfterLast("."))?.readText()
-
     public val isInstantiated: Boolean get() = ::instance.isInitialized
 
     override fun apply(runtime: Runtime<*>) {
-        runtime.load(ScriptContext(if (runtime.config.debuggable) debugScript else script, sourcePath ?: "$name.js"))
```

Comment:
Can you explain what the forward looking strategy is here since we're technically removing some functionality?

Reply:
two things here:

1. the functionality isn't working 😄 
2. once it does work, I think we'll remove the config option since it'll always be "debuggable" as long as you're pulling in the correct j2v8 dep, reg vs debuggable